poptile_rl/environment/engine.py

```diff
@@ -32,8 +31,7 @@ def pop_tile(self, row: int, col: int):
         self.generate_row()
 
         self.score += num_tiles ** 2
-        self.state_history.append(self.board.copy())
-        self.action_history.append((row, col))
+        self.last_action = row, col
```

Comment:
It'd be better to assign `(row, col)` instead of `row, col` for the better readability
```suggestion
        self.last_action = (row, col)
```